.changelog/96240bbb6b5341a0a5e40055b0222be0.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "96240bbb-6b53-41a0-a5e4-0055b0222be0",
+    "type": "release",
+    "description": "Add DSQL Auth Token Generator",
+    "modules": [
+        "feature/dsql/token"
```

Comment:
RDS has this in 
feature/RDS/auth right?

Could we align this and have it under 
```
feature/dsql/auth
```
?

Reply:
Correct. Moved the package to be aligned